server/api/pod/index.post.ts

```diff
@@ -0,0 +1,62 @@
+import { AbortError, ServiceError, fql } from 'fauna'
+
+// Body interface (read from POST request)
+interface Body {
```

Comment:
We might want to implement our type definitions for our Fauna entities in separate file(s).